test/manipulation.test.js

```diff
@@ -707,6 +707,59 @@ describe('manipulation', function () {
     it('should destroy filtered set of records');
   });
 
+  describe('deleteAll/destroyAll', function () {
+    beforeEach(function clearOldData(done) {
+      Person.deleteAll(done);
+    });
+
+    beforeEach(function createTestData(done) {
+      Person.create([
+        { name: 'John' },
+        { name: 'Jane' }
+      ], done);
+    });
+
+    it('should only delete instances that satisfy the where condition', function (done) {
+      Person.deleteAll({name: 'John'}, function (err, count) {
+        if (err) return done(err);
```

Comment:
lol, i see. I will change my tests in #502 to match for better error messages then. ;)